### DISCUSSION TITLE
Reorder shape attribute menu.

### DISCUSSION THREAD

**Discussion by pvaliyaveedan** [excalidraw/excalidraw#4059] [Q&A]
Is there a way to reorder the attribute menu for each shape? I would like to  move the duplicate and delete buttons to the top. Is it configurable?

**Comment by dwelle** [comment]
No. We might redesign this part though at some point. Are you talking about the application from a user perspective or a developer perspective (npm)?

**Reply by pvaliyaveedan** [reply]
Developer perspective. It would be great if the these are configurable.
